packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.test.ts

```diff
@@ -19,7 +19,7 @@ describe('Adapter is created as expected', () => {
     const mockCallClient = new MockCallClient() as unknown as StatefulCallClient;
 
     const mockCallAgent = new MockCallAgent();
-    const locator = { participantIds: ['some user id'] };
+    const locator = { groupId: 'some group id' };
```

Comment:
did this PR mean this needed changed?

Reply:
yes since we aren't GAing this locator and this is about whether the sound files are loaded to state